web-ui/src/components/search-results/SearchBirthdayAnniversaryResults.jsx

```diff
@@ -68,10 +67,10 @@ const SearchBirthdayAnniversaryResults = ({
   };
 
   const AnniversaryMap = () => {
-    if (searchAnniversaryResults.length > 0) {
+    if (anniversary && results.length > 0) {
```

Comment:
Nice to have change suggestion:

```js
results.length > 0
```

To simply:

```js
results.length
```

This works because `results.length` is truthy when the number of elements in the array is > 0. This will work for all instances where we check `.length > 0`, is idiomatic and a little more compact.

Reply:
I also prefer removing `> 0`. That was old code, but I'm making those changes everywhere that is done in this file.